src/js/main.js

```diff
@@ -366,6 +368,61 @@ Site.Earth = {
   init: function() {
     var _this = this;
 
+    if (_this.canHandleThree()) {
+      _this.initThreeEarth();
+    } else {
+      _this.initFlatEarth();
+    }
+
+  },
+
+  canHandleThree: function() {
+    var _this = this;
+
+    // init MobileDetect
+    var md = new MobileDetect(window.navigator.userAgent);
+
+    if (md.is('iOS') || md.is('Safari')) {
```

Comment:
This plugin isn't great (by its own admission) on desktop. I can't consistently check between Chrome and Safari using it. I was trying to invert the check to test if Chrome of FF so we could just apply the Three to those browsers, but no joy. 

I think a better approach is not to attempt to check browser but see if there are specific pieces of web tech that the Three scene we want uses that are not supported by Safari (on my Safari the scene renders just very very slowly and the audio analyser doesnt seem to return any values) Or perhaps just a desktop and mobile focused detector plugin is a better idea.